### ISSUE TITLE
Add issue template

### ISSUE THREAD

**Issue by MaxStalker** [leapdao/plasma-burner-wallet#224]
## Scope
For new contributors it's not clear how to issue should be filled according to bounty policy, which can lead to frustrating moments and also potential "issue stealing" 😱  

## Deliverables
Add templates for bug and feature issues

## Gain for the project
Issues will be filled in a consistent manner, which will help all parties - gardener, assignee and reviewer.

## Publicly visible delivery
Setup two templates for "bugs" and "features" for this repo

## Roles

bounty gardener: Max / 10%
bounty worker: Max / 75%
bounty reviewer: name / 15%

## Funded by
- EcoDev Circle (#ecosystem-development on LeapDAO Slack)

**Comment by vrde** [comment]
Just one note: if we use the [bounty template](https://hackmd.io/rujsCg2LRDKGsAHytZbi7w?view) as it is now, newcomers might be clueless on the different roles and funding circles.

The simplest solution I have in mind now is to leave those sections as optional, and let the gardener to take care of them.

**Comment by MaxStalker** [comment]
@TimDaub @johannbarbie I need some more rights in this repo to complete this task. Can any of you provide me full access for an hour? I promis to return them by midnight! 👑🏃‍♀️👡👗🎃🐭

**Comment by TimDaub** [comment]
> The simplest solution I have in mind now is to leave those sections as optional, and let the gardener to take care of them.

Agreed. I'd even go further and say: It's not necessary to have a template at all. We don't want absolute new-comers to be gardeners either, but only users that have contributed to LeapDAO for a month or so.

I'd go with the minimal template:

- Scope
- Deliverables
(- Gains)
- Roles

Everything else, I don't deem very useful.
